packages/graphql-server/src/plugins/useRedwoodRealtime.ts

```diff
@@ -19,35 +19,162 @@ export const liveDirectiveTypeDefs = print(
   astFromDirective(GraphQLLiveDirective)
 )
 
+export type LiveQueryStorageMechanism =
+  | RedisLiveQueryStore
+  | InMemoryLiveQueryStore
+
 export type RedwoodRealtimeOptions = {
-  liveQueries?: UseLiveQueryOptions
-  /**
-   * @description Subscriptions passed from the glob import:
-   * import subscriptions from 'src/subscriptions/**\/*.{js,ts}'
-   */
+  liveQueries?: {
+    store:
+      | 'in-memory'
+      | {
+          redis: {
+            channel?: string
+            publishClient: LiveQueryRedisClient
+            subscribeClient: LiveQueryRedisClient
+          }
+        }
+  }
   subscriptions?: {
+    /**
+     * @description Subscriptions passed from the glob import:
+     * import subscriptions from 'src/subscriptions/**\/*.{js,ts}'
+     */
```

Comment:
I think this description might need moved down slightly. It currently corresponds to the `store` when I think it's a description of the `subscriptions` property?

Reply:
Yup. I reworked come comments and included others.